neptune_lightgbm/impl/__init__.py

```diff
@@ -368,19 +380,25 @@ def create_booster_summary(
             ax.imshow(image.imread(s))
             ax.axis("off")
             trees_series.append(neptune.types.File.as_image(ax.figure))
-        results_dict["{}trees".format(visuals_path)] = neptune.types.FileSeries(trees_series)
+        results_dict[f"{visuals_path}trees"] = neptune.types.FileSeries(trees_series)
 
     if log_trees_as_dataframe:
         if isinstance(booster, lgb.Booster):
             df = booster.trees_to_dataframe()
-            html_df = neptune.types.File.as_html(df)
-            results_dict["trees_as_dataframe"] = html_df
-            if not df.empty and not html_df.content:
+            stream_buffer = BytesIO()
+            df.to_csv(stream_buffer, index=False)
+            results_dict["trees_as_dataframe"] = neptune.types.File.from_stream(
```

Comment:
@SiddhantSadangi I think this is a bit dangerous because currently streams have a limit of only 32MB. 

https://github.com/neptune-ai/neptune-client/issues/889#issuecomment-1117501542


Instead I would save it locally in a temp file and upload the file using .upload() because the limit of files uploaded in that way is much higher, about 5TB.

Docs: https://docs.neptune.ai/you-should-know/what-can-you-log-and-display#files